src/lsst/rsp/_discovery.py

```diff
@@ -0,0 +1,220 @@
+"""Service discovery within Nublado notebooks.
+
+These function suse a service discovery JSON file created by the Nublado
+controller when the user's lab was created for everything except obtaining
+InfluxDB credentials. This avoids a dependency on the Rubin Repertoire client,
+which may require a newer Python version than is available in the execution
+environment using lsst.rsp.
+"""
+
+from __future__ import annotations
+
+import json
+from dataclasses import dataclass
+from pathlib import Path
+from typing import Any
+
+import httpx
+
+from ._exceptions import (
+    DatasetNotSupportedError,
+    DiscoveryNotAvailableError,
+    InvalidDiscoveryError,
+    TokenNotAvailableError,
+    UnknownDatasetError,
+    UnknownInfluxDBError,
+    UnknownServiceError,
+)
+from .utils import get_access_token
+
+_DISCOVERY_PATH = Path("/etc/nublado/discovery/v1.json")
```

Comment:
Maybe plug this into .constants so we can monkeypatch it for testing?  Obviously we don't need to since you have this working but I feel like I usually regret it when I hardcode things like this when it comes time to do the test suite.